components/esp_ringbuf/ringbuf.c

```diff
@@ -1351,6 +1362,18 @@ void vRingbufferGetInfo(RingbufHandle_t xRingbuffer,
     portEXIT_CRITICAL(&pxRingbuffer->mux);
 }
 
+void vRingbufferUnblockRx(RingbufHandle_t xRingbuffer)
+{
+    Ringbuffer_t *pxRingbuffer = (Ringbuffer_t *)xRingbuffer;
+    configASSERT(pxRingbuffer);
+
+    // is the semaphore taken?
+    if (uxSemaphoreGetCount(rbGET_RX_SEM_HANDLE(pxRingbuffer)) == 0) {
```

Comment:
`rbGET_RX_SEM_HANDLE` doesn't seem to be declared anywhere.

Reply:
This code must have changed. @SoucheSouche, I'm pretty behind and stressed about my own work at the moment. If you are able to update it that would be a great help.

Reply:
sure no problem!

Reply:
I wonder if we should move this from the flags, and into its own variable. That way we are less likely to hit threading problems from multiple flags sharing the same memory location.

Reply:
^ I'm not familiar enough with ringbuf to make this judgement call.